assets/blocks/editor-components/toolbar-dropdown/index.js

```diff
@@ -0,0 +1,78 @@
+import {
+	Button,
+	Dropdown,
+	MenuGroup,
+	MenuItem,
+	NavigableMenu,
+} from '@wordpress/components';
+import classnames from 'classnames';
+
+/**
+ * @typedef {Object} DropdownOption
+ *
+ * @property {string} label Option label.
+ * @property {string} value Option value.
+ */
+/**
+ * Dropdown for the editor toolbar.
+ *
+ * @param {Object}           props
+ * @param {DropdownOption[]} props.options        Dropdown options.
+ * @param {string}           [props.optionsLabel] Options label.
+ * @param {Object}           props.icon           Icon for the toolbar.
+ * @param {string}           props.value          Current value.
+ * @param {Function}         props.onChange       Change function.
```

Comment:
I think that the arguments are a bit generic and the documentation don't specify what they actually do.

Reply:
Do you think this change is enough? https://github.com/Automattic/sensei/pull/3904/commits/6a94e1e961455f76505d77bde692d2e2d7ce1433